snippets/csharp/System.Net.Sockets/TcpClient/Overview/tcpclient.cs

```diff
@@ -65,22 +65,22 @@ static void Connect(String server, String message)
       // connected to the same address as specified by the server, port
       // combination.
       Int32 port = 13000;
-      TcpClient client = new TcpClient(server, port);
+
+      // Prefer using declaration to ensure the instance is Disposed later.
```

Comment:
> Prefer using declaration

"Prefer a using declaration"... otherwise it sounds like we should prefer to use a declaration rather than we should prefer to use a using declaration.